atlas/src/shared/components/Grid/Grid.tsx

```diff
@@ -1,21 +1,42 @@
 import React from 'react'
 import styled from '@emotion/styled'
 import useResizeObserver from 'use-resize-observer'
+import { useMediaQuery } from 'react-responsive'
 import { spacing } from '../../theme'
 
-const Container = styled.div<GridProps>`
+const toPx = (n: number | string) => (typeof n === 'number' ? `${n}px` : n)
+
+const LARGE_VIEWPORT_BREAKPOINT = toPx(2000)
```

Comment:
Maybe let's move that to `theme.breakpoints` so we don't have a bunch of breakpoints scattered across the source files

Reply:
Good Idea, moved.